packages/central-server/app/sync/CentralSyncManager.js

```diff
@@ -590,7 +590,7 @@ export class CentralSyncManager {
     return recordsForPull;
   }
 
-  async persistIncomingChanges(sessionId, deviceId, tablesToInclude) {
+  async persistIncomingChanges(sessionId, deviceId, tablesToInclude, isMobile) {
```

Comment:
I think it's fine to take from `debug_info`. That column was originally for debugging, but is now misnamed because we use it for proper stuff. You could rename it in this PR if you wanted